tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -224,6 +240,18 @@ public static Loader loader(String exportDir) {
     return new Loader(exportDir);
   }
 
+  /**
+   * Export a saved model.
+   *
+   * <p/>Returns a <code>Exporter</code> object for setting configuration options before actually
+   * saving the model.
+   *
+   * @param exportDir the directory path containing a saved model.
+   */
+  public static Exporter exporter(String exportDir) {
```

Comment:
Should we add a helper that takes an exportDir, a FunctionGraph and then performs all the necessary operations? Seems like there are a bunch of hoops to jump through if you just want to save a single `FunctionGraph`.

Reply:
I guess we could, what about making that endpoint available at the function level? Something like `function.save(exportDir)`

Reply:
I like that.